Sources/Basic/Lock.swift

```diff
@@ -53,6 +57,30 @@ public final class FileLock {
     ///
     /// Note: This method can throw if underlying POSIX methods fail.
     public func lock() throws {
+#if os(Windows)
```

Comment:
Minor note on indenting:

- `#if` statements should be indented two-space less than the regular code. See example: https://github.com/apple/swift-package-manager/blob/master/Sources/PackageDescription4/Package.swift#L68
- SwiftPM uses 4-space indent for regular code.
- When breaking a method into multiple lines, indent the parameters to the left of the call: Ex: https://github.com/apple/swift-package-manager/blob/master/Sources/Workspace/Workspace.swift#L370